app/components/UserProfile/useProfile.ts

```
@@ -0,0 +1,199 @@
+import { base } from 'viem/chains';
+import { formatAddress, stack } from '@app/lib/sdk/stack/client';
+import Unlock from '@app/lib/utils/Unlock.json';
+import { toast } from 'sonner';
+import type { ProfileState } from './types';
+
+const contractAddress = '0xf7c4cd399395d80f9d61fde833849106775269c6';
+
+type ProfileAction =
+  | { type: 'SET_LOADING'; payload: boolean }
+  | { type: 'SET_MEMBER_DATA'; payload: Account }
+  | { type: 'SET_NFT_DATA'; payload: any }
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)